Server/Game/Commands/Misc/AlertCommand.cs

```diff
@@ -14,21 +14,28 @@ internal class AlertCommand : ICommand
 
         public void OnCommand(ICommandExecutor executor, string label, ReadOnlySpan<string> args)
         {
-            if (args.Length >= 2)
+            if (executor is ClientSession session)
```

Comment:
This makes no sense, the console should be able to send alerts to other players.

Reply:
Whoopsies, my bad. :+1: